101-vm-with-standardssd-disk/azuredeploy.parameters.json

```diff
@@ -0,0 +1,44 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
```

Comment:
remove non-GEN values and rely on defaultValues in the template

Reply:
@bmoore-msft I don't understand your comment. Please give an example.